hazelcast-openshift-rhel/hazelcast-template.js

```diff
@@ -34,8 +35,16 @@
 					}
 				},
 				"spec": {
+					"volumes":[
+						{
+							"name":"hazelcast-storage",
+							"persistentVolumeClaim":{
+								"claimName":"hz-vc"
+							}
+						}
+					],
```

Comment:
I think this needs to go to documentation because first users will probably use the simplest approach. Adding volumes and customizing xml + custom jars would be more advanced usage.

The other option is to create another template with volume the volume option.